main.py

```diff
@@ -0,0 +1,8 @@
+from typing import Union
```

Comment:
Union is not being used in the code, let's take it out.

Reply:
Also, this file should go into an api folder, such that we have the structure:

api/main.py

Reply:
I can still find Union in here. Did you try removing it?

Reply:
Never mind. I see the change in #22